frappe_graphql/utils/generate_sdl/__init__.py

```diff
@@ -24,27 +24,33 @@
 
 
 def make_doctype_sdl_files(target_dir, app=None, modules=[], doctypes=[],
-                           ignore_custom_fields=False):
+                           ignore_custom_fields=False, disable_enum_selectdf=False):
```

Comment:
I think better to rename `disable_enum_selectdf` => `disable_enum_select_docfields` to be more readable

Reply:
@assemmarwan fixed 👍🏼